robot_skills/src/robot_skills/head.py

```diff
@@ -330,7 +328,7 @@ def detect_faces(self, stamp=False):
         else:
             return self._get_faces(image).recognitions
 
-    def get_best_face_recognition(self, recognitions, desired_label):
+    def get_best_face_recognition(self, recognitions, desired_label, probability_threshold=4.0):
```

Comment:
Needs more tuning or learning process needs to be more consistent. AMIGO was not able to recover me during following, because probability was lower than 3,5.

Reply:
@MatthijsBurgh , can you turn this into an issue in the backlog? Is too much to process before merging

Reply:
#399